projects/fmcdaq2/src/app/fmcdaq2.c

```diff
@@ -1068,7 +1075,7 @@ int main(void)
 		.dest_addr = (uintptr_t)ADC_DDR_BASEADDR
 	};
 	axi_dmac_transfer_start(fmcdaq2.ad9680_dmac, &transfer_rx);
-	status = axi_dmac_transfer_wait_completion(fmcdaq2.ad9680_dmac, 500);
+	status = axi_dmac_transfer_wait_completion(fmcdaq2.ad9680_dmac, 1000);
```

Comment:
This change is required so that correct operation is assured. If this modification is not performed, the DMA transfer does not complete and an error is generated. Should I move it to another PR?

Reply:
I created a separate commit for this modification.